src/collections/_documentation/workflow/integrations/global-integrations.md

```diff
@@ -899,41 +899,39 @@ Sentry owner or manager permissions are required to install this integration. Sl
 
 1. In Sentry, navigate to Organization Settings > **Integrations**.
 
-1. At the top of the page, you’ll see the available Global Integrations list, which includes Slack. Click **Install** to integrate Sentry with your Slack workspace(s).
+1. Once on the page, you’ll see the available Global Integrations list, which will include Slack. You can use the search bar to filter for Slack, if it doesn't appear at the top. 
```

Comment:
Suggest: "When you enter the page, the Global Integrations list displays, which includes Slack. Use the search bar to filter quickly for the Slack integration."

Reply:
@MeredithAnya can we make the references to "Global Integrations" just "integrations". Since now all of the integrations are in the list?